test/ReactWrapper-spec.js

```diff
@@ -465,6 +465,30 @@ describeWithDOM('mount', () => {
         ));
         expect(foundNotSpan).to.have.length(0);
       });
+
+      it('should return props object when props() is called', () => {
+        const SFC = function SFC({ selector }) {
+          return (
+            <div data-foo={selector}>Test SFC</div>
+          );
+        };
+
+        const selector = 'blah';
+        const wrapper = mount(<SFC data-foo={selector} />);
```

Comment:
I'm afraid I am not clear on what you mean.

As far as implementing the SFC correctly goes, do you mean passing properties that make sense? (I admit that was a sloppy use of an argument named `selector` I did there) Would the SFC used in lines 482-486 fit the bill?:
``` 
        const SFC = function SFC({ data }) {
           return (
             <div data-foo={data}>Test SFC</div>
           );
         };
```
And then as far as testing the component itself goes, what would be the right approach? Happy to make the changes, just want to understand before.